profusion/kernelCI-dashboard
# add docker services

## Description

Add docker-compose.yml creating 3 services:
- database service using postgresql;
- backend service for our django app;
- proxy service with nginx.

## Related Issues
- Closes #14 

## How to test it

Create secret files: 
```sh
mkdir -p backend/runtime/secrets
uuidgen > backend/runtime/secrets/postgres_password_secret
```
Startup the services with
 ```sh
 docker compose up --build -d
 ```

Make a request to the proxy, it should return the default Django content as we don't have any custom endpoints for now
 ```sh
 curl http://localhost:80
 ```
